### ISSUE TITLE
What LGPL License version is to be used?

### ISSUE THREAD

**Issue by Kim-SSi** [tomap/SevenZipSharp#12]
The readme reads that the project is under the LGPL v3.0.
But the license.txt is showing LGPL v2.1. It looks like the license.txt should be updated.
Which one is it meant to be?

Regards,
Kim

**Comment by tomap** [comment]
Hi,

I copied this repo from codeplex. So this licence issue comes from there. I'll update to v3.0, unless you prefer 2.0

Thomas

**Comment by Kim-SSi** [comment]
Hi,
As far as I can see it should be LGPL 3.0 reading the discussion on codeplex.
Are you the original author?
I would love if this was licensed as MIT - like a lot of .NET code. Or even dual licensed LGPL and MIT (or similar).

Kim

**Comment by tomap** [comment]
I'm not the original author, but 7zip itself is LGPL v2.1. So having sevenzipsharp in MIT or BSD won't help you much.
I'll update the license Asap

**Comment by Kim-SSi** [comment]
That is true. 7z.dll is LGPL and the SDK is public domain. It might be interesting to see how much of 7z.dll is in the SDK and could be replaced with that code compiled into a dll.
Thanks for the quick responses.
